misk-slack/src/main/kotlin/slack-api/SlackConfig.kt

```diff
@@ -0,0 +1,11 @@
+@file:Suppress("PropertyName") // We use snake_case to match Misk YAML.
```

Comment:
I don't think we use this in any other config classes so I think it can be removed.

Example: https://github.com/cashapp/misk/blob/c997e2dfeadf24c46b21153a830f8bd4f2278c7a/misk-jdbc/src/main/kotlin/misk/jdbc/DataSourceConfig.kt#L56